src/todo/AddOrEditList.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useCallback } from "react";
+import { connect } from "react-redux";
+import * as actions from "../store/actions/list";
+import {
+  Typography,
+  TextField,
+  FormLabel,
+  Container,
+  InputLabel,
+  FormControl,
+  Button,
+  Box,
+  makeStyles,
+} from "@material-ui/core";
+
+import Lists from "./Lists";
+import AddOrEditTodoInList from "./AddOrEditTodoInList";
+
+const useStyles = makeStyles((theme) => ({
+  inputList: {
+    marginTop: theme.spacing(2),
+    minWidth: 300,
+  },
+  btnAddList: {
+    marginTop: theme.spacing(2),
+    marginLeft: theme.spacing(1),
+  },
+  root:{
+    display: "flex",
+    flexDirection:'column',
+    [theme.breakpoints.up("sm")]: {
+      flexDirection:'row',
+     },
+   }
+}));
+
+const AddOrEditList = (props) => {
```

Comment:
if you think only few number of props are being passed to this component it's always better to import it like this
const AddOrEditList = ({list, addList}) => {}

Reply:
if not, then it's better to deconstruct all props at start of component
like this
const AddOrEditList = (props) => {
const {list, addList} = props;
}

Reply:
this way it's better to know the props without any time effort

Reply:
Yes sir done, already i know this.